doc/source/Getting_started/Troubleshooting.rst

```diff
@@ -159,26 +168,31 @@ Run your PyAEDT script. If the errors still persists, try the following:
     export ANSOFT_DEBUG_MODE=3
 
 
-Turn on the gRPC trace on the server side too:
+Enable the gRPC trace on the server:
 
 .. code:: console
 
     export GRPC_VERBOSITY=DEBUG
     export GRPC_TRACE=all
 
-Then start ansysedt.exe as gRPC server.
+Then start ansysedt.exe as a gRPC server and redirect the output.
 
 .. code:: console
 
-    ansysedt -grpcsrv 50051
+    ansysedt -grpcsrv 50051 > /path/to/file/server.txt
+
+The above command redirects the gRPC trace is
+to the file *server.txt*.
```

Comment:
```suggestion
to the``server.txt`` file.
```
Put filenames and paths in double back ticks.

Reply:
``server.txt`` is an example name, not a file type. The suggestion is confusing in that context.

Reply:
changed *server.txt* to ``server.txt``.